bot/modules/bartek.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import discord
+from discord.ext import commands
+
+from ..bot import DegeneratBot
+
+import csv
+import os
+from datetime import date
+
+PATH = "./data/polityka"
+
+class Bartek(commands.Cog):
+    def __init__(self, bot: DegeneratBot):
+        self.bot: DegeneratBot = bot
+
+        guild_id = os.getenv("GUILD_ID")
+        if not guild_id:
+            return logging.fatal("GUILD_ID environment variable not set!")
+
+        bartek_id = os.getenv("BARTEK_ID")
+        if not bartek_id:
+            return logging.fatal("BARTEK_ID environment variable not set!")
+
+        self.bartekCount = 0
+        self.blacklist = []
+        self.previousDate = date.today()
+
+        for file in os.listdir(PATH):
+            with open(f"{PATH}/{file}", "r") as f:
+                self.blacklist.extend(list(csv.reader(f, delimiter=";"))[0][:-1])
+
+        self.blacklist = set(self.blacklist)
```

Comment:
Instead of `blacklist` starting out as a `list`, it would be better to make it a `set` from the start, and use the `set.add` method, instead of `list.extend`